core/src/main/java/org/radargun/stages/lifecycle/ServiceStartStage.java

```diff
@@ -29,6 +34,18 @@
    @Property(doc = "If set to true, the slaves will not be started in one moment but the startup will be delayed. Default is true.")
    private boolean staggerSlaveStartup = true;
 
+   /**
+    * SYNC: It is the default behavior ( as it was before the scalability tests were introduced ).
+    * We add a delay between each node during the startup. The test only will start once we have all nodes up
+    * and running.
+    *
+    * ASYNC: It is a new behavior. It allows the node start without having to wait for all nodes be up and running.
+    * It allows the following: A test with 4 minutes and a delay of 1 minute between each node. This will simulate
+    * a scalability test.
+    */
+   @Property(doc = "If set to ASYNC, the slaves will startup will be delayed. Default is SYNC.")
```

Comment:
I think should refer to `delay-between-starting-slaves` property since that is what is used to delay the startup of the additional nodes

Reply:
We can't. `delay-between-starting-slaves` is just a delay when starting nodes. On the other hand, `ASYNC` is about allowing nodes to be able to start later without having the ` <cluster size="2" />` be fully filled.